manager/src/main/java/io/gingersnapproject/configuration/EagerConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package io.gingersnapproject.configuration;
+
+import java.util.Map;
+
+import javax.enterprise.event.Observes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.gingersnapproject.k8s.RuleEvents;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "gingersnap")
+public interface EagerConfiguration {
+   static final Logger log = LoggerFactory.getLogger(LazyConfiguration.class);
+
+   HotRod hotrod();
```

Comment:
I don't think it makes sense to have the `HotRod` configuration as part of `EagerConfiguration`.

If we keep the root `Configuration` as before, we can then just add methods for eager and lazy methods, e.g.:

```java
interface Configuration {
   HotRod hotrod();
  
   @WithName("lazy-rule")
   @WithDefault("")
   Map<String, LazyRule> lazyRules();
   
   @WithName("eager-rule")
   @WithDefault("")
   Map<String, EagerRule> eageRules();

}
```